src/DynamoCoreWpf/Views/Preview/WatchTree.xaml.cs

```diff
@@ -25,6 +26,15 @@ void WatchTree_Loaded(object sender, RoutedEventArgs e)
             _vm = this.DataContext as WatchViewModel;
         }
 
+        internal void SetWatchNodeProperties() 
+        {
+            resizeThumb.Visibility = Visibility.Visible;
+            this.Width = 200;
+            this.Height = 200;
```

Comment:
Do we need to respect `Configurations.MaxWatchNodeWidth` and `Configurations.MaxWatchNodeHeight` here? If not at least we should define these magic numbers as const or readonly somewhere

Reply:
Since we have the ability to expand the watch node manually to any desired size, we cannot set max height and width for this node now. 
I updated the PR to define these magic numbers as a const.